react/components/Modal/TopBar.js

```diff
@@ -13,6 +13,12 @@ const TopBar = props => {
     showCloseIcon,
   } = props
 
+  const handleKeyDown = event => {
+    if (event.key === 'Escape' || event.key === 'Esc') {
+      onClose()
+    }
+  }
```

Comment:
The correct thing is like this:

- the space key is an empty space (javascript and its jokes)

- Esc does not make sense to use, since esc has to be triggered at any time and not just when the focus is on the close button.

```suggestion
  const handleKeyDown = ({ key }) => {
    const SPACE = ' '
    const ENTER = 'Enter'

    if (key === SPACE || key === ENTER) {
      onClose()
    }
  }
```

Reply:
True! Updated